string.md

```diff
@@ -19,7 +19,7 @@ In CockroachDB, the following are synonyms of `STRING` and are implemented ident
 
 ## Length
 
-Length is always variable and unlimited for a `STRING` value. If you specify a fixed length by declaring `CHAR(n)` or `VARCHAR(n)` on a column, it will not be enforced.
+Length is always variable for a `STRING` value. If you specify a fixed length by declaring `CHAR(n)` or `VARCHAR(n)` on a column, it will not be enforced.
```

Comment:
Can we add another callout that a future version of CockroachDB will enforce the limit?